docs/CONFIG-VARS.md

```diff
@@ -43,22 +43,20 @@ Supported configuration variables are listed in the table below.  All variables
 | V4_DEPLOYMENT_OPERATOR_NAMESPACE | Namespace where the SAS Viya Deployment Operator should be installed  | string | "sasoperator" | false | Only applicable when V4_DEPLOYMENT_OPERATOR_SCOPE="cluster". | viya |
-| V4_DEPLOYMENT_OPERATOR_CRB | Name of the ClusterRoleBinding resource that is needed by the SAS Viya Deployment Operator | string | "sasoperator" | false | [Additional documentation](https://go.documentation.sas.com/doc/en/itopscdc/default/dplyml0phy0dkr/n137b56hwogd7in1onzys95awxqe.htm#p1arr91os91cg5n1tsmuamj6h10g) is available. | viya |
+| V4_DEPLOYMENT_OPERATOR_CRB | Name of the ClusterRoleBinding resource that is needed by the SAS Viya Deployment Operator | string | "sasoperator" | false | [Additional documentation](https://go.documentation.sas.com/doc/en/itopscdc/default/dplyml0phy0dkr/n137b56hwogd7in1onzys95awxqe.htm#p1arr91os91cg5n1tsmuamj6h10g) describing the resource is available. | viya |
 
 **Deployment Operator Notes:**
 
-* Currently, the viya4-deployment project does not support using the SAS Viya Deployment Operator in conjunction with a Long-Term Support: 2021.1 deployment that uses an alternate mirror repository (`V4_CFG_CR_URL`). If you are planning to deploy SAS Viya according to this specific scenario, SAS recommends using the orchestration tool to perform the installation instead.
-
-   This can be done by setting `V4_DEPLOYMENT_OPERATOR_ENABLED` to false.
-
+* Currently, the viya4-deployment project does not support using the SAS Viya Deployment Operator in conjunction with a Long-Term Support: 2021.1 deployment that uses an alternate mirror repository (`V4_CFG_CR_URL`). 
 
 * In a scenario where you have multiple SAS Viya deployments managed by a single cluster-wide deployment operator, uninstalling one of the SAS Viya deployments does not remove the SAS Viya Deployment Operator from the cluster. However, during the uninstallation workflow, if no SAS Viya deployments that are managed by the cluster-wide deployment operator are detected, the SAS Viya Deployment Operator is also removed.
 
+* If you are running this project using Ansible directly on your workstation, we require Docker to be installed and the executing user should be able to access it. This is required to use the sas-orchestration command. See [ansible usage](user/AnsibleUsage.md#Preparation).
 
-* If you are running this project using Ansible directly on your workstation, we require Docker to be installed and the executing user should be able to access it. This is required to use the Orchestration Tooling CLI see [ansible usage](user/AnsibleUsage.md#Preparation)
+* Using the sas-orchestration command to perform a Viya deployment is only applicable for cadences 2022.12 and beyond. For previous cadences please use the Deployment Operator to perform your deployments. 
```

Comment:
Talked with @supear, we would like the last bullet reads: Using the sas-orchestration deploy command to perform a Viya deployment is only applicable for SAS Viya 2022.12 and later. For previous releases, use the SAS Viya Deployment Operator to perform your deployments.

Reply:
@sayeun, the bullet point has been updated